config/manifests/bases/jira-service-desk-operator.clusterserviceversion.yaml

```diff
@@ -1,60 +0,0 @@
-apiVersion: operators.coreos.com/v1alpha1
```

Comment:
should this file be deleted?

Reply:
No, this needs to be re-generated based on the latest structure